src/Components/CreateImageWizard/steps/Registration/components/ManualActivationKey.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useRef } from 'react';
+
+import {
+  Button,
+  Content,
+  ContentVariants,
+  FormGroup,
+  FormGroupLabelHelp,
+  Popover,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { ACTIVATION_KEYS_URL } from '../../../../../constants';
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeActivationKey,
+  changeOrgId,
+  selectActivationKey,
+  selectOrgId,
+} from '../../../../../store/wizardSlice';
+import { ValidatedInput } from '../../../ValidatedInput';
+
+const ManualRegistrationPopover = () => {
+  const labelHelpRef = useRef(null);
+
+  return (
+    <Popover
+      triggerRef={labelHelpRef}
+      headerContent='About Activation Keys & Organization ID'
+      position='right'
+      minWidth='30rem'
+      bodyContent={
+        <Content>
+          <Content component={ContentVariants.p}>
+            Activation keys assist you in registering and configuring systems.
+            Metadata such as role, system purpose, and usage can be
+            automatically attached to systems via an activation key, and
+            monitored with Subscription Watch.
+          </Content>
+          <Content component={ContentVariants.p}>
+            The Organization ID is the numeric identifier for your organization
+            and is separate from your account number. Your organization&apos;s
+            activation keys and organization ID are displayed on the Activation
+            Keys page in the Hybrid Cloud Console.
+          </Content>
+          <Button
+            component='a'
+            target='_blank'
+            variant='link'
+            icon={<ExternalLinkAltIcon />}
+            iconPosition='right'
+            isInline
+            href={ACTIVATION_KEYS_URL}
+          >
+            Vie activation keys in Hybrid Cloud Console
```

Comment:
```suggestion
            View activation keys in Hybrid Cloud Console
```

otherwise looks good to me!

Reply:
Agh flip